arm-recoveryservices/2016-06-01/swagger/vaults.json

```diff
@@ -615,8 +684,7 @@
       ],
       "properties": {
         "properties": {
-          "$ref": "#/definitions/VaultProperties",
-          "x-ms-client-flatten": true
+          "$ref": "#/definitions/VaultProperties"

```

Comment:
I understand @dragonfly91 implied that there is a bug at their service end. If this indeed is the structure that the service expects, this shouldn't be a blocker but we highly recommend that this be revisited and fixed.